hypertrace-view-generator/build.gradle.kts

```diff
@@ -31,7 +31,7 @@ dependencies {
   implementation("org.hypertrace.core.datamodel:data-model:0.1.9")
   implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.15")
 
-  implementation("org.hypertrace.traceenricher:enriched-span-constants:0.1.23")
+  implementation("org.hypertrace.traceenricher:enriched-span-constants:0.1.23") // TODO: upgrade its version once the related PR is merged in trace-enricher
```

Comment:
This is dependent on https://github.com/hypertrace/hypertrace-trace-enricher/pull/41 . Once that PR is merged, I can upgrade its version.
This is the reason why the builds have failed currently